src/xform_to_datalog/arcs_manifest_tree/particle_spec.h

```diff
@@ -62,14 +71,22 @@ class ParticleSpec {
       std::string name,
       std::vector<raksha::ir::TagCheck> checks,
       std::vector<raksha::ir::TagClaim> claims,
-      std::vector<raksha::ir::Edge> edges)
+      std::vector<HandleConnectionSpec> handle_connection_specs)
       : name_(std::move(name)), checks_(std::move(checks)),
-        claims_(std::move(claims)), edges_(std::move(edges)) { }
+        claims_(std::move(claims)) {
+    for (HandleConnectionSpec &handle_connection_spec :
```

Comment:
Is this tested?

Reply:
Yes. All of the tests in `particle_spec_test.cc` use a textproto string to provide the ParticleSpec test input. `ParticleSpec::CreateFromProto` uses this constructor to build a `ParticleSpec` after constructing the input collections from the proto, and thus exercises this constructor.